### PR TITLE
One of my favourite hug quotes

### PR DESCRIPTION
**Pull request by mishmanners**
Add to the status. One of my favourite hug quotes ever.

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -39,4 +39,5 @@
    "They're a 10, but they get cranky if they don't get their daily hug.",
    "If I text you \"🤗\" it means I'm sending you a great humongous hug.",
    "hey (with the intention of giving you a long tight hug because you deserve it)",
+   "Hugs are a great gift 🤗 0one size fits all, and no one minds if you return it",
```
Comment (achasveachas) [inline statuses.py:42]: Looks like there's a typo (`0one size...`).

Once it's fixed I will merge it.

**Review by achasveachas — CHANGES_REQUESTED** [review]

**Review by achasveachas — APPROVED** [review]
Looks great! Thanks @mishmanners 🤗 

Let me know if you have a twitter handle so I can give you a shout out.

**Comment by mishmanners** [comment]
Hey thank you @achasveachas. I do have a Twitter handle - MishManners
[See profile](https://twitter.com/mishmanners)

**Comment by mishmanners** [comment]
Did you get my message @achasveachas 😄 would love the shout out so I can retweet!

**Comment by achasveachas** [comment]
Oh, my bad @mishmanners! I must have missed the notification!

**Pull request merged**